ontology/observable/observable.ttl

```diff
@@ -10637,6 +10781,20 @@ observable:partitionOffset
 	rdfs:range xsd:integer ;
 	.
 
+observable:partyDevice
+	a owl:ObjectProperty ;
+	rdfs:label "partyDevice"@en ;
+	rdfs:comment "The IMEI of the device connected to the cell tower."@en ;
```

Comment:
This documentation comment confuses me as to what the `rdfs:range` is intended to be.  `observable:ObservableDevice`?  The range of the `observable:IMEI` property (which happens to be a datatype property with range `xsd:string`, implying this property would be `xsd:string` as well)?